doc/image-from-registry.md

```diff
@@ -0,0 +1,42 @@
+# Creating a registry image with a VM disk
+The purpose of this document is to show how to create registry image containing a Virtual Machine image that can be import into a PV.
+
+## Prerequisites
+The VM disk image should be called 'disk.img' and placed in the root folder.
+
+## Create an image with Buildah
```

Comment:
could you add a few words about Buildah and refer to its documentation?

Reply:
Done